src/core/Akka.API.Tests/verify/CoreAPISpec.ApproveCore.DotNet.verified.txt

```diff
@@ -1409,15 +1409,10 @@ namespace Akka.Actor
     }
     public sealed class Nobody : Akka.Actor.MinimalActorRef
     {
-        public static Akka.Actor.Nobody Instance;
+        public static readonly Akka.Actor.Nobody Instance;
         public override Akka.Actor.ActorPath Path { get; }
         public override Akka.Actor.IActorRefProvider Provider { get; }
         public override Akka.Util.ISurrogate ToSurrogate(Akka.Actor.ActorSystem system) { }
-        public class NobodySurrogate : Akka.Util.ISurrogate
```

Comment:
Never needed to be part of the public API - just the `ISurrogate` interface is enough.